installers/vmware/kickstart-esxi.go

```diff
@@ -308,18 +301,13 @@ kickstartpishellcmd=$(custom_data "['kickstart']['postinstall_shell_cmd']")
 
 # Kickstart postinstall supplemental config URL
 if [ "$kickstartpiurl" != "null" ]; then
-        echo "Using supplemental kickstart postinstall URL: $kickstartpiurl"
-        if wget -q -s $kickstartpiurl; then
-                wget -q $kickstartpiurl -O /tmp/ks-postinstall-sup.sh
-                echo "========Begin execution of supplemental postinstall kickstart"
-                chmod +x /tmp/ks-postinstall-sup.sh && /tmp/ks-postinstall-sup.sh
-                echo "========End execution of supplemental postinstall kickstart"
-        else
-                echo "ERROR: Custom kickstart postinstall URL is NOT accessible!"
-                exit 1
-        fi
+	echo "Using supplemental kickstart postinstall URL: $kickstartpiurl"
+	wget -q $kickstartpiurl -O /tmp/ks-postinstall-sup.sh
+	echo "========Begin execution of supplemental postinstall kickstart"
+	chmod +x /tmp/ks-postinstall-sup.sh && /tmp/ks-postinstall-sup.sh
+	echo "========End execution of supplemental postinstall kickstart"
```

Comment:
Ditto. Seems like we went off the track a little here, can you apply this suggestion instead?

```suggestion
	echo "Using supplemental kickstart postinstall URL: $kickstartpiurl"
	if wget -q "$kickstartpiurl" -O /tmp/ks-postinstall-sup.sh
		echo "========Begin execution of supplemental postinstall kickstart"
		chmod +x /tmp/ks-postinstall-sup.sh && /tmp/ks-postinstall-sup.sh
		echo "========End execution of supplemental postinstall kickstart"
	else
		echo "ERROR: Custom kickstart postinstall URL '$kickstartpiurl' is NOT accessible!"
		exit 1
	fi
```